src/main/java/com/internet/shop/models/User.java

```diff
@@ -0,0 +1,46 @@
+package com.internet.shop.models;
```

Comment:
```suggestion
package com.internet.shop.model;
```

let's name packages in singular

Reply:
Done